api/api.go

```diff
@@ -3,33 +3,23 @@ package api
 import (
 	"context"
 
-	"github.com/ONSdigital/dp-identity-api/config"
+	cognitoclient "github.com/ONSdigital/dp-identity-api/cognitoclient"
```

Comment:
Not sure this alias is needed as the package name is the same as the alias. Also consider renaming the package as it _"stutters"_ i.e. `client.client` which is against the Go naming convention.  Go is very opinionated about naming things which can present an annoying burden to work around. `cognito` seems like a reasonable package name.